server/forms/step.ts

```diff
@@ -9,7 +9,12 @@ const isDateString = (value: any): boolean => {
   return moment(value, moment.ISO_8601).isValid()
 }
 
+const join = (values: Array<any>, seperator: string): string => {
+  return values.join(seperator)
+}
+
 jsonLogic.add_operation('isDateString', isDateString)
+jsonLogic.add_operation('join', join)
```

Comment:
It might be easier to just define join as Array.prototype.join rather than define our own function IE: 
`jsonLogic.add_operation('join', Array.prototype.join)`